frontend/src/components/settings/ManageUserDialog.vue

```diff
@@ -240,17 +244,27 @@ export default {
     /**
      * Update user and emit success/fail event.
      */
-    async updateUser() {
+    async updateUser(user) {
       try {
-        //TODO - complete when API is ready
-        //this.user.organizationId = this.userInfo.organizationId
-        //const response = await ApiService.apiAxios.post(ApiRoutes.USER + '/update', this.user)
+        const response = await ApiService.apiAxios.post(ApiRoutes.USER + '/update', this.user)
```

Comment:
Response is declared but not use. Your linter should pick this up.

Reply:
Fixed.